### DISCUSSION TITLE
How to change contrast of selected items

### DISCUSSION THREAD

**Discussion by LogWell** [linuxmint/discussions#46] [Ideas]
After selecting an item (_bookmarks_6_11_23.html_), jump to the **Extra Panel**, where the display contrast of the item is very low.

![20230619_104230](https://github.com/linuxmint/discussions/assets/34391447/a1d5fc1b-95c3-4faf-b725-7b095a0859fa)
![20230619_104236](https://github.com/linuxmint/discussions/assets/34391447/752c9cbf-2266-484f-8ef8-e2d379b5d33e)

**Comment by marcus125** [comment]
Hi,
I found a fix for this and have submitted a [pull request](https://github.com/linuxmint/nemo/pull/3261). Note that the problem only occurs with the themes:
- Adwaita
- Adwaita-dark
- HighContrast

It might take a while for the fix to be in the official copy of linux mint, so if you want to fix it in the meantime, either use a different theme or create `~/.config/gtk-3.0/gtk.css` and add `.nemo-window .nemo-inactive-pane .view:selected { background-color: #3584e4; }`, then reopen the files program (this bit of css adds a blue background to what is selected).
